src/components/Button/Button.jsx

```diff
@@ -2,11 +2,12 @@ import PropTypes from 'prop-types'
 import cn from 'classnames'
 import './Button.scss'
 
-const Button = ({ onClick, label, className, disabled, block, ...rest }) =>
+const Button = ({ onClick, label, className, disabled, block, invisible, ...rest }) =>
```

Comment:
You can remove invisible.
It's better to use 
```
<>
  {  isButtonShouldBeShown && <Button>...</Button> }
</>
```

Reply:
Done